### PR TITLE
[MemTracker] Make some MemTracker have reasonable parent MemTracker to eazily to trace. Add show level of MemTracker

### LINKED ISSUE #5782: [Enhance] Make MemTracker More Reasonable
### 2 Make Each MemTracker can be trace:
        
The current memory consumption is mainly concentrated on the following operations:

- query
- load
- compaction
- schema change

But it's really difficult to Identify the memory consumption caused by that particular operation, such as three queries. We don't know which query occupies the most memory.

So we need a way to identify it

- query ：each memtracker of same instance the label have same instance id
- load：same as above
- compaction ： each memtracker of same tablet of compaction the label have same tablet id
- schema change： each memtracker of same tablet of schema change the label have old tablet id - new tablet id

### PR DESCRIPTION
**Pull request by HappenLee**
1 Make some MemTracker have reasonable parent MemTracker not the root tracker
2 Make each MemTracker can be eazily to trace.
3 Add show level of MemTracker to reduce the MemTracker show in the web page to have a way to control show how many tracker in web page.

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [x] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #5782 ) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] If these changes need document changes, I have updated the document
- [x] Any dependent changes have been merged

### CODE REVIEW

File: be/src/common/config.h [hunk 1]
```diff
@@ -597,6 +597,8 @@ CONF_mInt32(zone_map_row_num_threshold, "20");
 //    Trace = 6
 CONF_Int32(aws_log_level, "3");
 
+// default level of MemTracker to show in webpage
```
Comment (morningman) [inline be/src/common/config.h:600]: Explain whether this value is higher and more detailed or lower and more detailed.

File: be/src/olap/memtable.cpp [hunk 2]
```diff
@@ -40,7 +40,7 @@ MemTable::MemTable(int64_t tablet_id, Schema* schema, const TabletSchema* tablet
           _slot_descs(slot_descs),
           _keys_type(keys_type),
           _row_comparator(_schema),
-          _mem_tracker(MemTracker::CreateTracker(-1, "MemTable:" + std::to_string(tablet_id), parent_tracker)),
+          _mem_tracker(MemTracker::CreateTracker(-1, "MemTable", parent_tracker)),
```
Comment (morningman) [inline be/src/olap/memtable.cpp:43]: Why remove tablet id?

File: be/src/runtime/tablets_channel.cpp [hunk 3]
```diff
@@ -34,7 +34,7 @@ std::atomic<uint64_t> TabletsChannel::_s_tablet_writer_count;
 TabletsChannel::TabletsChannel(const TabletsChannelKey& key,
                                const std::shared_ptr<MemTracker>& mem_tracker)
         : _key(key), _state(kInitialized), _closed_senders(64) {
-    _mem_tracker = MemTracker::CreateTracker(-1, "TabletsChannel:" + key.to_string(), mem_tracker);
+    _mem_tracker = MemTracker::CreateTracker(-1, "TabletsChannel", mem_tracker);
```
Comment (morningman) [inline be/src/runtime/tablets_channel.cpp:37]: Why remove key?

File: be/src/olap/memtable.cpp (hunk 2 above)
Comment (HappenLee) [reply be/src/olap/memtable.cpp:43]: Beacause the tracker will use the parent label id

File: be/src/runtime/tablets_channel.cpp (hunk 3 above)
Comment (HappenLee) [reply be/src/runtime/tablets_channel.cpp:37]: The same to up

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**